File: javascript/dardo.js
const d1 = parseFloat(prompt("Digite a primeira distancia"));
const d2 = parseFloat(prompt("Digite a segunda distancia"));
const d3 = parseFloat(prompt("Digite a terceira distancia"));


let maior = d1;

if (d2 > d1) {
    maior = d2;
} else if (d3 > d1) {
    maior = d3;
}

console.log("Maior distancia = %.2f", maior);

File: python/dardo.py
print("Digite as tres distancias:")
distancia1 = float(input())
distancia2 = float(input())
distancia3 = float(input())

maior = distancia1

if distancia2 > maior:
    maior = distancia2
elif distancia3 > maior:
    maior = distancia3

print(f"MAIOR DISTANCIA = {maior:.2f}")
